packages/react-next/src/components/Slider/Slider.scss

```diff
@@ -0,0 +1,262 @@
+    }
+  }
+}
+
+@mixin activeSectionActiveStyles {
+  .activeSection {
+    background-color: var($semanticColorsInputBackgroundCheckedHovered);
+    @include high-contrast {
+      background-color: Highlight;
+    }
+  }
+}
+
+// Styles for when the slider is hovered or active
+@mixin inactiveSectionActiveStyles {
+  .inactiveSection {
+    background-color: var($semanticColorsInputPlaceholderBackgroundChecked);
+    @include high-contrast {
+      border-color: Highlight;
+    }
+  }
+}
+
+// Styles for when the slider is hovered or active
+@mixin thumbActiveStyles {
+  .thumb {
+    border: 2px solid var($semanticColorsInputBackgroundCheckedHovered);
+    @include high-contrast {
+      bordercolor: Highlight;
+    }
+  }
+}
+
+@mixin slideBoxActiveZeroTickStyles {
+  .zeroTick {
+    background-color: var($semanticColorsInputPlaceholderBackgroundChecked);
+    @include high-contrast {
+      background-color: Highlight;
+    }
+  }
+}
+
+.root {
+  @include fonts-medium;
+  user-select: none;
+
+  &.vertical {
+    margin-right: 8;
+  }
+
+  &:not(.disabled) {
+    &:active {
+      @include activeSectionActiveStyles();
+      @include inactiveSectionActiveStyles();
+      @include thumbActiveStyles();
+      @include slideBoxActiveZeroTickStyles();
+    }
+
+    &:hover {
+      @include activeSectionHoverStyles();
+      @include inactiveSectionActiveStyles();
+      @include thumbActiveStyles();
+      @include slideBoxActiveZeroTickStyles();
+    }
+  }
+}
+
+.titleLabel {
+  padding: 0;
+}
+
+.container {
+  display: flex;
+  flex-wrap: nowrap;
+  align-items: center;
+  &.vertical {
+    flex-direction: column;
+    height: 100%;
+    text-align: center;
+    margin: 8px 0;
+  }
+}
+
+.slideBox {
+  @include focus-border();
+  background: transparent;
+  border: none;
+  flex-grow: 1;
+  line-height: 28px;
+  display: flex;
+  align-items: center;
+  height: 28px;
+  width: auto;
+  padding: 0 8px; // Make room for thumb at ends of line
+
+  &.vertical {
+    height: 100%;
+    width: 28px;
+    padding: 8px 0; // Make room for thumb at bottom of line
+  }
+}
+
+.thumb {
+  border-width: 2px;
+  border-style: solid;
+  border-color: var($semanticColorsSmallInputBorder);
+  border-radius: 10px;
+  box-sizing: border-box;
+  background: var($semanticColorsInputBackground);
+  display: block;
+  width: 16px;
+  height: 16px;
+  position: absolute;
+  transform: translateX(-50%);
+
+  // TODO either remove or implement for RTL transform: translateX(50%);
+
+  &:not(.vertical) {
+    top: -6px;
+  }
+
+  &.vertical {
+    left: -6px;
+    margin: 0 auto;
+    transform: translateY(8px);
+  }
+
+  &.disabled {
+    border-color: var($semanticColorsDisabledBorder);
+    @include high-contrast {
+      bordercolor: GrayText;
+    }
+  }
+
+  /**
+   * TODO Implement animations
+   * .showTransitions && {
+   * transition: `left ${AnimationVariables.durationValue3} ${AnimationVariables.easeFunction1}`;
+   * };
+   */
```

Comment:
FYI I'm going to add animation constants in my PR [#13370](https://github.com/microsoft/fluentui/pull/13370/files#diff-108d44f6cfaa5f4557efce5da2f23aed). Feel free to comment there if you think there's a better way to add the constants.